uitools/import/Esri/ArcGISRuntime/Toolkit/esri_arcgisruntime_toolkit_view.qrc

```diff
@@ -1,24 +1,24 @@
 <RCC>
-  <!-- This section contains all view components (shared by C++ and QML APIs) -->
```

Comment:
Not sure how it was removed, didn't mean to delete it.
I usually modify the `qrc` files using a different text editor because Qt Creator doesn't show the xml format

Reply:
It's hidden but there! 
<img width="485" alt="image" src="https://user-images.githubusercontent.com/42203545/149989231-c0aad225-f967-4f47-a393-fa7aeb95a732.png">